prime-router/settings/prod/0150-CA-Add-FullELR-SaphireStage-Receiver.yml

```diff
@@ -14,7 +14,7 @@
       topic: "full-elr"
       customerStatus: "testing"
       translation: !<HL7>
-        schemaName: "metadata/hl7_mapping/receivers/STLTs/CA/CA"
```

Comment:
Can we split out this change?  It doesn't seem related and would we not need to sync this to the environment?

Reply:
Agreed, opened #11468